src/api/spec/policies/comment_policy_spec.rb

```diff
@@ -67,4 +70,39 @@
       it { expect(subject).not_to permit(other_user, comment_on_request) }
     end
   end
+
+  # rubocop:disable RSpec/RepeatedExample
+  # This cop is currently not recognizing the permissions block as separate test
+  permissions :update? do
+    it 'Not logged users cannot edit comments' do
```

Comment:
I know that you followed what is already done above in this file, but it's not a so great example to follow. The wording is quite poor there. We could improve it too, but it's not in the scope of this PR. For now, I suggest going with something like this:

```suggestion
    it 'an anonymous user cannot update comments' do
```